R/estimateRichness.R

```diff
@@ -181,7 +183,7 @@ NULL
 #' @export
 setGeneric("estimateRichness",signature = c("x"),
         function(x, abund_values = "counts",
-                  index = c("observed", "chao1", "ace", "hill"),
+                  index = c("ace", "chao1", "hill", "observed"),
```

Comment:
These are just rearrangements. What is their purpose?

Reply:
Yes, they are just reordered in alphabetical order. The purpose is that now they are in some kind of order. For me it looks nicer. Also, as a user I would prefer that small things like these are considered

But I can revert that change